src/accessories/CoverAccessory.ts

```diff
@@ -10,6 +10,10 @@ import { BaseAccessory } from "./BaseAccessory";
 import { TuyaDevice } from "../api/response";
 
 export class CoverAccessory extends BaseAccessory {
+  public target = 0;
+  public position = 0;
```

Comment:
Please change these [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) to enums